index.rst

```diff
@@ -4,9 +4,326 @@ Monitor coupling in M1M3 VMS system and TMA drives together
+   In this technote we investigate the coupling between M1M3 and Telescope Mount Assembly (TMA) using the Vibration Monitor System (VMS) data.
+   There are two parts in the technote. The first one is a general study where we analyse the vibrations recorded by the VMS during various slews in elevation and azimuth.
+   We also analyze the VMS data when the TMA is in tracking mode. 
 
-Add content here
-================
+   In the second part, we perform a detailed unvestigation of a peculiar event occuring on June 28th, 2023 around 1:08 am where a strong noise were reported during and after an elevation slew.
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+   Associated JIRA ticket: `SITCOM-784 <https://rubinobs.atlassian.net/browse/SITCOM-784>`_
+
+The M1M3 Vibration Monitor System (VMS)
+=======================================
+
+The M1M3 cell is equipped with 3 accelerometers measuring accelerations along 3 axes (x, y and z) that all together are composing the M1M3 Vibration Monitor System or VMS.
+The following figure shows the location of the 3 accelerometers on the mirror cell.
+
+.. figure:: /_static/images/accelerometers-location-annotated.png
+   :name: accel_location
+   :target: _images/accelerometers-location-annotated.png
+   :alt: Location of the VMS accelerometers on the mirror cell
+
+   Location of the VMS accelerometers on the mirror cell (red circles)
+
+The next figure shows the mount details of the 3 VMS accelerometers on the mirror cell. We can see that they are rotated in such a way that their x, y and z axes are oriented in the same way. 
+The 3 accelerometers are measuring the same thing on 3 different locations of the mirror cell.
+
+.. figure:: /_static/images/accelerometers_mount_details.png
+   :name: accel_details
+   :target: _images/accelerometers_mount_details.png
+   :alt: Mount details of the 3 VMS accelerometers 
+
+   Mount details of the 3 VMS accelerometers 
+
+The VMS channels are readout with a sampling frequency of 200 Hz the corresponding datasets are written in hdf5 format on the vms-data.cp.lsst.org server at the summit. These large files (~1.3 GB)
+will eventually be copied on the LFA but in the mean time, some of them have been copied to /scratch/users/b/boutigny/vmsdata/ and some of them have been converted to a parquet format holding a more convenient
+pandas dataframe.
+
+Note that VMS data also exists for M2 and for the Rotator.
+
+Study of vibrations recorded by the VMS during slews
+====================================================
+
+For this study we have selected large amplitude azimuth et elevation slews occuring on December 21st, 2023.
+
+The following plot shows the acceleration along the x axis recorded by VMS accelerometer #3 over a 24 hour-period on 2023/12/21. The other accelerometers / axes show a similar behavior. 
```

Comment:
It is not an extraordinary case, I have seen few occurrences of such vibration events. In fact it would be interesting to make a global study but this is outside the scope of this technical note. 
The idea is more to say : "we see instances of shaking events and we have the tools ready to investigate them in case it occurs again with the glass mirror"

Reply:
OK! I think it would be good to have this clearly stated. So that the actual conclusion is that this exists, the tools are available, and a possible plan is to make a global study.